tips_n_tricks/week3/map-collisions.md

```diff
@@ -0,0 +1,151 @@
+Notice though, we need to give it x and y **cell coordinates**. That refers to the x and y coordinates on the bottom left of the map editor when you hover over a sprite on the map.
+
+![](mget.png)
+
+Because each cell coordinate refers to 1 sprite and each sprite is 8x8 pixels, the sprite that the mouse is pointed at in the above picture would be at the **pixel coordinates** of (8, 112) or (**1 x 8**, **14 x 8**).
+
+`mget` returns the sprite number given the cell x and y coordinates, so `mget(1, 14)` would give us `1` in the picture above, because the sprite at coordinate (1,14) is 1.
+
+
+### [fget(n, f)](http://pico-8.wikia.com/wiki/Fget)
+
+Now that we know what sprite is at a particular x and y coordinate, we can check if that sprite has a flag set.   
+
+**Sprite flags** are those little circles below the color and size pickers in the Sprite Editor.
+
+![](fget.png)
+
+The sprite selected in the above picture has **flag 0** turned on.
+If we look at the documentation, we see that `fget(n, f)` takes a sprite number first and then the flag number second. 
+That means to get the flag for the sprite in the above picture, we'd do `fget(1, 0)`, looking at sprite number 1 to see if flag 0 is set.
+
+Flags can be used any way you'd like but in the case of our map collisions, we can say "if a sprite has flag 0 turned on, the player can't go through it. It's a solid object."
+
+That way I can put other sprites on the map and if flag 0 is off, they'll just be part of the background and the player can just walk through it.
+
+![](fget2.png)
+
+*Sprite 5's flag 0 is off. That means the player can walk through it.*
+
+### Putting it all together
+
+#### 1. `mget`
+Using `mget`, we can get the sprite number of each sprite on the map. I've drawn the output of `mget` on the actual sprites here so we can see them easier.
+
+![](map_mget.png)
+
+#### 2. `fget`
+
+Now if we use `fget` with those sprite numbers, we'll be able to tell if that place on the map is "solid" or not. I've colored the sprite numbers <span style="color:red;">red</span> if their flag 0 is on.
+
+![](map_fget.png)   
+\* Notice, the grasses' sprite numbers aren't red
+
+#### 3. Player Location
+
+Alright, that's great and all but we don't need every sprite and every flag on the map, we just need to know the sprite and flag that the character is on so that we can tell if she's colliding or not.
+
+How do we do that though? Our character's x and y coordinates represent pixels, but `mget` takes x and y map cell coordinates!
+
+The Player's x and y coordinates on the screen can be anywhere on the screen, that's 128x128, but the map's x and y cell coordinates count the sprites, not the pixels. If we cut the screen into chunks of 8 pixels each, each one of those 8x8 chunks is a sprite and is one cell coordinate. In order to *divide* something into chunks... we **divide** it! So 128px / 8px per sprite == **16 sprites**. Our screen can only hold 16x16 sprites. 
```

Comment:
"on the screen" is repeated twice in the opening sentence.

Since you just talked about how the player coords are pixel-coords and mget takes cell-coords, it might be worth re-wording some of this to focus on the actual problem the reader faces: translating between two related coordinate systems via a ratio. Right now it's kind of unclear what you're getting at, especially since the conclusion ("We can use that calculation to figure out which map cell our charater is on.") is not actually in this paragraph.